Libraries/Image/__tests__/resolveAssetSource-test.js

```diff
@@ -159,6 +160,34 @@ describe('resolveAssetSource', () => {
       });
     });
   });
+  
+  describe('bundle was loaded from file on Android', () => {
+    beforeEach(() => {
+      NativeModules.SourceCode.scriptURL =
+        '/sdcard/Path/To/Simulator/main.bundle';
```

Comment:
If loaded from the file, does RN Android report its `scriptURL` without `file://` prefix? If so, we should fix that instead of adding complicated parsing and unnecessary `if` statements.